v2/getbakersrewardperiod.go

```diff
@@ -0,0 +1,19 @@
+package v2
+
+import (
+	"context"
+
+	"github.com/Concordium/concordium-go-sdk/v2/pb"
+)
+
+// GetBakersRewardPeriod retrieves all bakers in the reward period of a block.
+//
+// This endpoint is only supported for protocol version 6 and onwards.
+func (c *Client) GetBakersRewardPeriod(ctx context.Context, req isBlockHashInput) (_ pb.Queries_GetBakersRewardPeriodClient, err error) {
```

Comment:
Could we please add a wrapper for GetBakersRewardPeriodClient that uses `BakerId` type wrapper we have defined in types.go?